superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterBar.tsx

```diff
@@ -242,22 +232,19 @@ const FilterBar: React.FC<FiltersBarProps> = ({
 
   const handleFilterSelectionChange = (
     filter: Pick<Filter, 'id'> & Partial<Filter>,
-    extraFormData: ExtraFormData,
-    currentState: CurrentFilterState,
+    filtersState: DataMask,
   ) => {
+    // @ts-ignore
```

Comment:
This should probably be cleaned up in a follow-up PR (it seems we need to refine a few types in `superset-ui`)